src/range-slider/index.ts

```diff
@@ -0,0 +1,372 @@
+			'aria-invalid': invalid === true ? 'true' : null,
+			'aria-readonly': readOnly === true ? 'true' : null,
+			'aria-label': isSlider1 ? minimumLabel : maximumLabel,
+			type: 'range',
+			min: `${min}`,
+			max: `${max}`,
+			step: `${step}`,
+			readonly: readOnly,
+			required,
+			disabled,
+			name: isSlider1 ? minName : maxName,
+			onblur: prepareCallback((prop, e1, e2) => this._genericCallback(prop, e1, e2), this.properties.onBlur),
+			onclick: prepareCallback((prop, e1, e2) => this._genericCallback(prop, e1, e2), this.properties.onClick),
+			onfocus: prepareCallback((prop, e1, e2) => this._genericCallback(prop, e1, e2), this.properties.onFocus),
+			onchange: prepareCallback((prop, e1, e2) => this._genericChangeCallback(prop, e1, e2), this.properties.onChange),
+			oninput: prepareCallback((prop, e1, e2) => this._genericChangeCallback(prop, e1, e2), this.properties.onInput),
+			onkeydown: this._onKeyDown,
+			onkeypress: this._onKeyPress,
+			onkeyup: this._onKeyUp,
+			onmousedown: this._onMouseDown,
+			onmouseup: this._onMouseUp,
+			ontouchstart: this._onTouchStart,
+			ontouchend: this._onTouchEnd,
+			ontouchcancel: this._onTouchCancel,
+			classes: [this.theme(css.input), fixedCss.nativeInput]
+		};
+	}
+
+	protected renderOutput(minValue: number, maxValue: number, percentValue: number[]): DNode {
+		const {
+			output,
+			outputIsTooltip = false
+		} = this.properties;
+
+		const outputNode = output ? output(minValue, maxValue) : `${minValue}, ${maxValue}`;
+
+		// output styles
+		let outputStyles: { left?: string; top?: string } = {};
+		if (outputIsTooltip) {
+			outputStyles = { left: `${Math.round((percentValue[0] + (percentValue[1] - percentValue[0]) / 2) * 100)}%` };
+		}
+
+		return v('output', {
+			classes: this.theme([css.output, outputIsTooltip ? css.outputTooltip : null]),
+			for: this._widgetId,
+			styles: outputStyles,
+			tabIndex: -1 /* needed so Edge doesn't select the element while tabbing through */
+		}, [outputNode]);
+	}
+
+	render(): DNode {
+		const {
+			disabled,
+			widgetId = this._widgetId,
+			invalid,
+			label,
+			labelAfter,
+			labelHidden,
+			max = 100,
+			min = 0,
+			readOnly,
+			required,
+			theme,
+			showOutput = false
+		} = this.properties;
+		const focus = this.meta(Focus).get('root');
+		let { minValue = min, maxValue = max } = this.properties;
+
+		minValue = Math.max(minValue, min);
+		maxValue = Math.min(maxValue, max);
+
+		const slider1Percent = (minValue - min) / (max - min);
+		const slider2Percent = (maxValue - min) / (max - min);
+
+		const slider1Size = slider1Percent + (slider2Percent - slider1Percent) / 2;
+		const slider2Size = 1 - slider1Size;
+
+		const size = this.meta(Dimensions).get('root');
+
+		const slider1Focus = this.meta(Focus).get('slider1');
+		const slider2Focus = this.meta(Focus).get('slider2');
+
+		const slider1 = v('input', {
+			...this._getInputProperties(true),
+			key: 'slider1',
+			value: `${minValue}`,
```

Comment:
Is there any drawback to setting `max` here and `min` on the next input to the `maxValue` and `minValue`? I think it could be useful, but up to you :)

Reply:
Yeah, so setting the max/min like that changes the position of the thumb within the slider (since the total range of the slider changes) and I couldn't get things to smoothly move when sliding the sliders around